x-pack/plugin/core/src/main/java/org/elasticsearch/license/LicenseOverrides.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.license;
+
+import org.elasticsearch.common.hash.MessageDigests;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
```

Comment:
Is there a reason we're using old school `Date` rather than java 8 `LocalDate` or `ZonedDateTime` ?

I suspect we want to use ZDT (specified in UTC)

Reply:
Old, bad habits and not doing much with the objects. Thanks, updated to use `ZonedDateTime`s (by way of a `LocalDateTime` so we can ensure we'll get an exception if the specified datetime doesn't exist in the given zone).